CHANGES.rst

```diff
@@ -12,6 +12,7 @@ Improvements
 
 - Include co-authors in abstract list columns and spreadsheet exports (:pr:`5605`)
 - Include speakers in abstract list columns and spreadsheet exports (:pr:`5615`)
+- Expose room location in room booking's UI (:pr:`5622`)
```

Comment:
you can credit yourself here since you aren't a team member (see the `thanks :user:...` snippets used for some other external contributions)